src/components/landing/Welcome.jsx

```diff
@@ -1,5 +1,12 @@
 const Welcome = () => {
-  return <div>Image of Bourns</div>;
+  return (
+    <img
```

Comment:
this shouldn't be included in this pull request, remove the changes in this file and put them into a new branch, make a new pull request with the new branch